vpn/tunnel.go

```diff
@@ -40,7 +63,13 @@ func NewTunnel(
 		speaker:         *(s),
 		ctx:             ctx,
 		logger:          logger,
+		clientLogger:    slog.Make(),
```

Comment:
If we don't use the protocol for logging, we'll want to use the logger that got passed in, rather than just swallowing those logs.